packages/postcss-svgo/src/index.js

```diff
@@ -75,6 +70,9 @@ function minifyPromise (decl, getSvgo, opts) {
                     });
                 })
                 .catch(error => {
+                    if (String(error).startsWith('Error in parsing SVG')) {
+                        return;
+                    }
                     throw new Error(`${PLUGIN}: ${error}`);
```

Comment:
Let's do warning here too, as in original PR

Reply:
sorry, didn't notice the PR had another commit. rebased changes.